tests/python/test-1-vector.py

```diff
@@ -136,17 +136,31 @@ def test_104(ceed_resource):
 
     assert a[3] == -3.14
 
+# -------------------------------------------------------------------------------
+# Test creation, setting, reading, restoring, and destroying of a vector using
+#   CEED_MEM_DEVICE
+# -------------------------------------------------------------------------------
+
 
 def test_105(ceed_resource):
-    """Modification of reshaped array"""
-    ceed = libceed.Ceed(ceed_resource)
+    # Skip test for non-GPU backend
```

Comment:
@jeremylt  I'm not able to test this even on Summit, but if you are and it works fine for you, what would you think to add it to the vector class' tutorial, specifying that that would work only if one has built libCEED with GPU support?

Reply:
Sometimes installing Numba with CUDA support can be touchy. I can work on adding that. Would we want to add it in this PR or separately after we update PyPi?

Reply:
Seems out of scope for libCEED itself, unless you're talking about CI.

Reply:
Bad spacing/layout on my part. Two thoughts.
1) Numba is touchy - explaining why there might be issues in @valeriabarra's Summit env
2) Do we want to add it in this PR - referencing adding GPU vectors to the tutorial

Reply:
2. It should be in some tutorial by the time we submit to SciPy.

Reply:
Sure. And I know the cell I want to add. I just wasn't sure if anyone cared about if it would be before or after updating PyPi.

Reply:
Added. We can cherry pick the commit if we want to wait until after PyPi is updated.

Reply:
Thanks Jeremy!

Reply:
What is it needed for the PyPi update?

Reply:
I've done it in the past. Run `make wheel` to build the wheel in a docker container. I can give you access to upload to PyPI, but it does not allow any modifications after upload so you need to be confident it's the way you want. There is test.pypi.org that can be used to check.

Reply:
I'm not sure I could be extremely confident on which way would be the way we want the wheel to be uploaded. Not to mention I've never even created a docker container myself. Sorry. I wanted to help, but perhaps it is better and faster if I don't embark in this.